Source/WebCore/platform/Site.cpp

```diff
@@ -52,4 +52,9 @@ bool Site::matches(const URL& url) const
     return url.protocol() == m_protocol && m_domain.matches(url);
 }
 
+String Site::string()
+{
+    return makeString(m_protocol, "://"_s, m_domain.string());
```

Comment:
Should empty site actually mean no partition (i.e. we return `""` here?)

Reply:
Yes, good catch.